mealpy/mealpy.py

```diff
@@ -190,5 +193,30 @@ def execute_reserve_meal(restaurant, reservation_time, city):
 @click.argument('restaurant')
 @click.argument('reservation_time')
 @click.argument('city')
-def reserve(restaurant, reservation_time, city):
+def reserve(restaurant, reservation_time, city):  # pragma: no cover
     execute_reserve_meal(restaurant, reservation_time, city)
+
+
+@cli.group(name='list')
+def cli_list():  # pragma: no cover
+    pass
+
+
+@cli_list.command('cities', short_help='List available cities.')
+def cli_list_cities():  # pragma: no cover
+    cities = [i['name'] for i in get_cities()]
+    print('\n'.join(cities))
+
+
+@cli_list.command('restaurants', short_help='List available restaurants.')
+@click.argument('city')
+def cli_list_restaurants(city):  # pragma: no cover
+    restaurants = [i['restaurant']['name'] for i in get_schedules(city)]
+    print('\n'.join(restaurants))
```

Comment:
I haven't tried this, but would this and `meals` flood the screen with many lines of text? I was thinking of a search feature to find meals or restaurants that is tolerant to typos.

Reply:
Yes it would and that's by-design. I would not be in favour of implementing or adding on tools to process the data. It's trivial to pipe the output to `grep` or `fzf`.

I don't see an interactive CLI ever being useful for this project, in cost-benefit. The initial setup of the project requires some information and your suggestions would help those 5 minutes. But once user is fully configured, they never use nor care for any of features.